bundle/config/mutator/python/python_mutator.go

```diff
@@ -193,6 +193,11 @@ func (m *pythonMutator) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagno
 		return nil
 	}
 
+	// Don't run any arbitrary code when DATABRICKS_CLI_RESTRICTED_CODE_EXECUTION is set.
+	if os.Getenv("DATABRICKS_CLI_RESTRICTED_CODE_EXECUTION") != "" {
```

Comment:
Let's use the `DATABRICKS_BUNDLE_` prefix and move the lookup to `bundle/env`.

Rejection execution is specific to DABs here.

The package defines accessors for other bundle environment variables.